src/Modal/ModalPortal.tsx

```diff
@@ -74,7 +95,7 @@ export default class ModalPortal extends React.Component {
     } = this.props;
 
     return () => (
-      <ReactTransitionGroup transitionAppear>
```

Comment:
`ReactTransitionGroup` doesn't take any `transitionAppear` prop. Should be useless